site/ko/beta/guide/saved_model.ipynb

```diff
@@ -0,0 +1,1183 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "nAzRHR0UT4hv"
+   },
+   "outputs": [],
+   "source": [
+    "!saved_model_cli show --dir /tmp/module_with_signature --tag_set serve --signature_def serving_default"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "0B25WsscTZoC"
+   },
+   "outputs": [],
+   "source": [
+    "imported = tf.saved_model.load(\"/tmp/module_with_signature\")\n",
+    "signature = imported.signatures[\"serving_default\"]\n",
+    "assert [3.] == signature(x=tf.constant([3.]))[\"output_0\"].numpy()\n",
+    "imported.mutate(tf.constant(2.))\n",
+    "assert [6.] == signature(x=tf.constant([3.]))[\"output_0\"].numpy()\n",
+    "assert 2. == imported.v.numpy()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "_gH91j1IR4tq"
+   },
+   "source": [
+    "하나의 시그니처를 내보냈고 그 키는 \"serving_default\"를 기본으로 설정했습니다. 여러 시그니처를 내보내려면 딕셔너리형으로 전달합니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "6VYAiQmLUiox"
+   },
+   "outputs": [],
+   "source": [
+    "@tf.function(input_signature=[tf.TensorSpec([], tf.string)])\n",
+    "def parse_string(string_input):\n",
+    "  return imported(tf.strings.to_number(string_input))\n",
+    "\n",
+    "signatures = {\"serving_default\": parse_string,\n",
+    "              \"from_float\": imported.signatures[\"serving_default\"]}\n",
+    "\n",
+    "tf.saved_model.save(imported, \"/tmp/module_with_multiple_signatures\", signatures)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "8IPx_0RWEx07"
+   },
+   "outputs": [],
+   "source": [
+    "!saved_model_cli show --dir /tmp/module_with_multiple_signatures --tag_set serve"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "sRdSPjKFfQpx"
+   },
+   "source": [
+    "`saved_model_cli`는 커맨드 라인에서 SavedModel을 직접 실행할 수도 있습니다."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "hcHmgVytfODo"
+   },
+   "outputs": [],
+   "source": [
+    "!saved_model_cli run --dir /tmp/module_with_multiple_signatures --tag_set serve --signature_def serving_default --input_exprs=\"string_input='3.'\"\n",
+    "!saved_model_cli run --dir /tmp/module_with_multiple_signatures --tag_set serve --signature_def from_float --input_exprs=\"x=3.\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "WiNhHa_Ne82K"
+   },
+   "source": [
+    "## 가져온 모델의 세부 사항 조정하기\n",
+    "\n",
+    "변수 객체는 사용할 수 있으며 가져온 함수를 통해 역전파할 수 있습니다."
```

Comment:
You can translate imported as verb or object name.
I like the latter in this case and want to avoid repeating '~수 있다'.
I suggest '변수 객체는 사용할 수 있으며 가져온 함수를 통해 역전파할 수 있습니다' --> '변수 객체가 사용 가능하므로 imported 함수를 통해 역전파할 수 있습니다'